pytest_tests/testsuites/acl/test_acl.py

```diff
@@ -178,3 +180,149 @@ def test_basic_acl_readonly(self, wallets, client_shell, read_only_container, fi
                 shell=client_shell,
                 cluster=self.cluster,
             )
+
+    @allure.title("Test basic ACL IR and STORAGE rules compliance")
+    def test_basic_acl_ir_storage_rules_compliance(self, wallets, public_container, file_path):
+        """
+        Test basic ACL IR and STORAGE rules compliance.
+        """
+        endpoint = self.cluster.default_rpc_endpoint
+
+        user_wallet = wallets.get_wallet()
+        ir_wallet = wallets.get_ir_wallet()
+        storage_wallet = wallets.get_storage_wallet()
+
+        cid = public_container
+
+        with allure.step("Add test objects to container"):
+            owner_object_oid = put_object_to_random_node(
+                user_wallet.wallet_path, file_path, cid, shell=self.shell, cluster=self.cluster
+            )
+
+        with allure.step("Check IR and STORAGE rules compliance"):
+            assert not can_put_object(
```

Comment:
Can we use ```check_full_access_to_container (robot/resources/lib/python_keywords/container_access.py)``` or ```check_no_access_to_container``` here? or some modification of it maybe?

Reply:
They've already been put there. ```check_full_access_to_container``` looks pretty much the same as these conditions here with some variations. But ok, we can leave it as is

Reply:
> looks pretty much the same

But it's not full access here